pkg/authorization/controller/authorizationsync/origin_to_rbac_test.go

```diff
@@ -0,0 +1,545 @@
+package authorizationsync
```

Comment:
\<nit pick mode on\>
It's convenient to have a controller and accompanying test along with it, can you split this into 4 files, please 🙏 
\</nit pick mode off\>